docs/feeds.md

```diff
@@ -0,0 +1,106 @@
+
+#  Implementing Feed Actions
+
+The *feed action* is a normal OpenWhisk *action*, but it should accept the following parameters:
+* **lifecycleEvent**: one of 'CREATE', 'DELETE', 'PAUSE', or 'UNPAUSE'
+* **triggerName**: the fully-qualified name of the trigger which contains events produced from this feed.
+* **authKey**: the Basic auth credentials of the OpenWhisk user who owns the trigger just mentioned
+
+The feed action can also accept any other parameters it needs to manage the feed.  For example the cloudant changes feed action expects to receive parameters including *'dbname'*, *'username'*, etc.
+
+When the user creates a trigger from tthe CLI with the **--feed** parameter, the system automatically invokes the feed action with the appropriate parameters.
+
+For example,assume the user has created a `mycloudant` binding for the `cloudant`
+package with their username and password as bound parameters.  When the user issues the following command from the CLI,
+
+`wsk trigger create T --feed mycloudant/changes -p dbName myTable`
+
+then under the covers the system will do something equivalent to:
+
+`wsk action invoke mycloudant/changes -p lifecycleEvent CREATE -p triggerName T -p authKey <userAuthKey> -p password <password value from mycloudant binding> -p username <username value from mycloudant binding> -p dbName mytype`
+
+The feed action named *changes* takes these parameters, and is expected to take whatever action is necessary to set up a stream of events from Cloudant, with the appropriate configuration, directed to the trigger *T*.    
+
+For the Cloudant *changes* feed, the action happens to talk directly to a *cloudant trigger* service we've implemented with a connection-based architecture.   We'll discuss the other architectures below.
+
+A similar feed action protocol occurs for `wsk trigger delete`.    
+
+We have not yet implemented pause and unpause, but they will be similar.
+
+# Implementing Feeds with Hooks
+
+Setting up a feed via a hook is *by far* the easiest way to prototype a new feed,
+if the even producer supports a webhook
+facility.
+
+
+With this method there is _no need_ to stand up any persistent service outside of OpenWhisk.  All feed management happens naturally though stateless OpenWhisk *feed actions*, which negotiate
+directly with a third part webhook API.
+
+When invoked with `CREATE`, the feed action simply installs a webhook for some other service, asking the remote service to POST notifications to the appropriate `fireTrigger` URL in OpenWhisk.
+
+The webhook should be directed to send notifications to a URL such as:
+`POST /namespaces/{namespace}/triggers/{triggerName}`
```

Comment:
I would add that the JSON body of the HTTP POST will be the parameters use to fire the trigger, that will be forward to the actions associated via rules.